app/assets/stylesheets/_users.scss

```diff
@@ -0,0 +1,131 @@
+// Place all the styles related to the users controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+body {
+  font-family: candara;
+}
+
+$light-grey : lighten(#b2beb5, 10%);
+form {
+  border-right: 2px solid $light-grey;
+}
+
+div.row.container {
+  box-shadow: none;
+  width: 60%;
+  margin: auto;
+  top: 0; left: 0; bottom: 0; right: 0;
```

Comment:
Property 'left' should be placed on own line<br>Property 'bottom' should be placed on own line<br>Property 'right' should be placed on own line